_projects/community-share.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Community Share
+date: 2015-06-01
+link: https://github.com/communityshare/communityshare
+intro: Come help with
+image: /assets/images/communityshare.png
+lead: Josh Schachter & Ben Reynwar
+contact: http://www.communityshare.us/contact/
```

Comment:
Two points:

 - HTML _is_ valid Markdown
 - I agree and think that using `[Community Share Contact Page](url…)` is a good idea, though the HTML might be necessary if we want the `_blank` target